tests/e2e/README.md

```diff
@@ -107,6 +107,8 @@ E2E tests have multiple options available while running them as follows:
 * `--sidecar_injector_hub <hub>` - Image hub for the Sidecar Injector (default: environment $HUB)
 * `--sidecar_injector_tag <tag>` - Image tag for the Sidecar Injector (default: environment $TAG)
 * `--sidecar_injector_file <file>` - Sidecar injector YAML file name (default: istio-sidecar-injector.yaml)
+* `--cni_hub <hub>` - Image hub for the CNI plugin: (default: environment $HUB)
```

Comment:
Consider making it a bit specific such as:
--istio_cni_hub <hub> - Image hub for the Istio CNI plugin: ....
Similar comments with cni_tag.